Scuti/Scripts/UI/UIManager.cs

```diff
@@ -22,14 +22,25 @@ public class UIManager : MonoBehaviour {
 
         public GameObject EventSystemObject;
 
+        public GameObject firstSelected;
```

Comment:
@Jpablogs8 what does this do in UIManager?

Reply:
This can already be deleted. Initially I had it here so that when scuti was started it would have an initial selection and this would be handled by UIManager. Then I thought as there are many screens and popups it would complicate this.

Reply:
Great, please remove!